podcasts/Analytics/AnalyticsEvent.swift

```diff
@@ -335,6 +335,7 @@ enum AnalyticsEvent: String {
     case playerTabSelected
     case playerShowNotesLinkTapped
     case playerChapterSelected
+    case playerPodcastLinkTapped
```

Comment:
I wonder if this should be `PodcastNameTapped` to mirror the `episode_detail_podcast_name_tapped` event in the Episode Detail page when tapping the name.